massa-consensus-worker/src/controller.rs

```diff
@@ -301,29 +298,14 @@ impl ConsensusController for ConsensusControllerImpl {
     }
 
     fn subscribe_new_blocks_headers(&self, sink: SubscriptionSink) {
```

Comment:
While you are refactoring I don't really understand what this functions are doing. Is not suppose to be the API that start the thread when it receive someone subscribing ? And the modules only broadcast ? 

So in your module you just need to know about a tokio sender and the ws related stuff is contained in API.

Reply:
May be the module `massa-ws` could be removed. see the comment above please

Reply:
It's not related to the comment above. It's about using the websocket things only in API and only have sender in the other modules.

Reply:
done